tsconfig.json

```diff
@@ -1,10 +1,3 @@
 {
-  "extends": "./tsconfig.build.json",
-  "compilerOptions": {
-    "baseUrl": ".",
-    "paths": {
-      "@eth-optimism/*": ["packages/*/src", "integration-tests/*/test"]
-    },
```

Comment:
Let's keep this. OK with removing the others.

Reply:
This affects certain packages behaviour. For example, running `tslint` in `packages/contracts` will lead to linting of `packages/smock` I _think_ because it interprets it as part of the source code. I haven't fully debugged that example in particular but removing this line fixes it.

Do we need that feature? I prefer building the packages and using TypeScript's [Project References](https://www.typescriptlang.org/docs/handbook/project-references.html).

Reply:
KK